question28/js/validations.js

```diff
@@ -0,0 +1,80 @@
+const submitButton = document.getElementById('js-submit-button');
```

Comment:
IMO
The module should be reusable and self-contained.

Therefore, I think that retrieving and changing values here would complicate the dependency.
It would be more appropriate to obtain it at the source of use.

Reply:
Thank you for your review! I learned a lot. Functions that need to get a value were not modularized and returned to their respective files.Please let me know if my perception is different.:bow:

Fixed commit :https://github.com/csakyo/homework/pull/42/commits/03e50c027820f69fdbf6d1a864c84a759c5db67f